pytorch_lightning/core/hooks.py

```diff
@@ -64,19 +64,17 @@ def on_predict_end(self) -> None:
     def on_pretrain_routine_start(self) -> None:
         """Called at the beginning of the pretrain routine (between fit and train start).
 
-        - fit
```

Comment:
Remember to update the CHANGELOG and any mentions in our docs

Reply:
Will do. I will make this change in the subsequent PR.

Reply:
Let's make the documentation change in this PR. It is very often subsequent PR gets lost.